app/assets/javascripts/event_staff.js

```diff
@@ -10,6 +11,20 @@ $(document).ready(function(){
           $(".event_staffs").append(data);
           $(".add_staff_field").val("");
         });
+=======
+
+  $(document).ready(function(){
+
+    var isExistingOnPage = function (userId) {
+      var parent = $("#event_staffs").children(),
+          isOnPage = false;
+
+      for (var i = 0; i < parent.length; i += 1) {
+        if(parent[i].dataset.id === userId) {
+        isOnPage = true;
+        break;
+        };
+>>>>>>> 5d4aa35... jcbajkbcka
```

Comment:
Expected an identifier and instead saw '>>>'.<br>Expected an operator and instead saw '>>>'.<br>Unexpected '5'.<br>Expected an operator and instead saw '>'.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.